libs/core/src/scss/base/_variables.scss

```diff
@@ -89,18 +89,11 @@ $border-radius-round: 999px !default;
 ****************************************************************************/
 $breakpoints: (
   small: 321px,
-  medium: 721px,
-  large: 1025px,
-  xlarge: 60em,
-  xxlarge: 80em,
+  medium: 632px,
+  large: 768px,
+  xlarge: 992px,
+  xxlarge: 1440px,
```

Comment:
I dont think we want this 'breaking' change, where e.g. medium breakpoint is redefined to 632px. It should be 769px, right? 

I made the mistake of updating the value to 768px in a different PR, but it should have been 769px.